src/OpenSheetMusicDisplay/OSMDOptions.ts

```diff
@@ -239,6 +239,10 @@ export interface IOSMDOptions {
      * This works across instruments- If all instruments have subsequent measures with nothing but rests, multirest measures are generated
      */
     autoGenerateMutipleRestMeasuresFromRestMeasures?: boolean;
+    cursorsOptions?: CursorOptions[];
+    /**
+     * Defines multiple simultaneous cursors. If left undefined the standard cursor will be used.
+     */
```

Comment:
The docstring needs to be above the variable, not below.

Reply:
done